File: task2_diamond_buggy.py
# Task 2: Buggy Diamond (with AI allowed)
n = 5
for i in range(1, n+1):
    print(" " * (n - i) + "*" * (2 * i - 1))

# Buggy lower half
for i in range(n-1):
    print(" " * (n - i) + "*" * (2 * i - 1))

File: task2_diamond_buggy.java
// Task 2: Buggy Diamond in Java (with AI allowed)
// Final output should look like:

//     *
//    ***
//   *****
//  *******
// *********
//  *******
//   *****
//    ***
//     *

public class Task2DiamondBuggy {
    public static void main(String[] args) {
        int n = 5;
        for (int i = 1; i <= n; i++) {
            System.out.println(" ".repeat(n - i) + "*".repeat(2 * i - 1));
        }
        
        }
    }
}